### DISCUSSION TITLE
Start time

### DISCUSSION THREAD

**Discussion by MechEng70** [kudrykv/latex-yearly-planner#18] [Ideas]
Any chance that you have the day start at 5am?  Due to covid, we are starting many meetings at 6am.  

Also, do you have a buy me a coffee link? Or something?

**Comment by kudrykv** [comment]
I can build one. You need to tell me which planner you want, and I will adjust the hours.

You can try Paypal: hello@bgnfu7re.me

**Reply by pbasch** [reply]
Hi! Love your planner. I would love it to go from 6AM to 6PM... (Sunday start of week)
I will try your link above for the "buy me coffee" thing (I'm brand new to public Github, so this is new to me; been using it at work for LaTeX projects) Thanks!

**Reply by pbasch** [reply]
Oh, it's in Russian! Will you instead just take my grateful thanks?

**Reply by pbasch** [reply]
I guess I should be very specific -- I would like a week starting Sunday, and AM/PM hours, and the day going from 6AM to 6PM. I hope that's clear. Thanks for the beautiful work!

**Reply by pbasch** [reply]
Wow, always another detail... For the Supernote A5x. And, hey, as long as I'm asking for customizations, Instead of "The best thing that happened today", could you put, "What made me laugh today"
I hope this isn't too much. I know this is a volunteer project. Thanks!
[You know, maybe I should look at your LaTeX code, if I can work it...]

**Reply by kudrykv** [reply]
Hi @pbasch!

I'll generate one.

> it's in Russian!

It's Ukrainian :)

> 6AM to 6PM

Do you want to have a little calendar on daily pages? Otherwise, A5X fits 16 hours in the schedule.

**Comment by MechEng70** [comment]
Hi, sorry, I just saw this... Here is the one that I am currently using: rm2.mos.lined.default.sun.dailycal.2022.pdf

Edit: It did fail, however, my bank did ask if it was fraud or not.  I let them know it was not.  Please check in the next couple of days to see if you received it.  thanks

**Reply by kudrykv** [reply]
Here you go—[rm2.mos.lined.default.sun.dailycal.2022.pdf](https://github.com/kudrykv/latex-yearly-planner/files/7821282/rm2.mos.lined.default.sun.dailycal.2022.pdf)

**Reply by MechEng70** [reply]
Thank you.  I checked the bank and it does not seem to have gone through.  Do you see it pending on your end?

Any other means to send contributes?

**Reply by kudrykv** [reply]
I guess it didn't go through.
That's fine, though. I'll need to figure out another way.
Thank you for being interested in this project!

**Comment by pbasch** [comment]
I have Venmo and PayPal. Does that work? And what is the contribution you
expect for your time?

*Peter Basch *

**Reply by kudrykv** [reply]
Unfortunately both don't work here. Don't worry though, it's fine.

**Comment by pbasch** [comment]
I like the little calendar. I can't imagine needing 16 hours... I never
want to work that hard! Bad enough I need it to start at 6am...

I have to say, I am incredibly impressed at your LaTeX skills.

*Peter Basch *

**Reply by kudrykv** [reply]
You should keep your fun activities in the calendar, too. A planner isn't solely for work!

Cool, I'll generate one soon

**Reply by kudrykv** [reply]
I've realized that I'm missing some info:
- there are two different headers for planners—MOS and Breadcrumb;
- there are two patterns available—lined and dotted;
- 12 hours doesn't fit on the page — it has to be 11 hours.

Because of that, I've generated a bunch of them so that you can pick one. Please tell with which you end up, haha.
[sn_a5x.breadcrumb.dotted.default.ampm.sun.dailycal.2022.0617.pdf](https://github.com/kudrykv/latex-yearly-planner/files/7838704/sn_a5x.breadcrumb.dotted.default.ampm.sun.dailycal.2022.0617.pdf)
[sn_a5x.breadcrumb.dotted.default.ampm.sun.dailycal.2022.0718.pdf](https://github.com/kudrykv/latex-yearly-planner/files/7838705/sn_a5x.breadcrumb.dotted.default.ampm.sun.dailycal.2022.0718.pdf)
[sn_a5x.breadcrumb.lined.default.ampm.sun.dailycal.2022.0617.pdf](https://github.com/kudrykv/latex-yearly-planner/files/7838706/sn_a5x.breadcrumb.lined.default.ampm.sun.dailycal.2022.0617.pdf)
[sn_a5x.breadcrumb.lined.default.ampm.sun.dailycal.2022.0718.pdf](https://github.com/kudrykv/latex-yearly-planner/files/7838707/sn_a5x.breadcrumb.lined.default.ampm.sun.dailycal.2022.0718.pdf)
[sn_a5x.mos.dotted.default.ampm.sun.dailycal.2022.0617.pdf](https://github.com/kudrykv/latex-yearly-planner/files/7838709/sn_a5x.mos.dotted.default.ampm.sun.dailycal.2022.0617.pdf)
[sn_a5x.mos.dotted.default.ampm.sun.dailycal.2022.0718.pdf](https://github.com/kudrykv/latex-yearly-planner/files/7838710/sn_a5x.mos.dotted.default.ampm.sun.dailycal.2022.0718.pdf)
[sn_a5x.mos.lined.default.ampm.sun.dailycal.2022.0617.pdf](https://github.com/kudrykv/latex-yearly-planner/files/7838711/sn_a5x.mos.lined.default.ampm.sun.dailycal.2022.0617.pdf)
[sn_a5x.mos.lined.default.ampm.sun.dailycal.2022.0718.pdf](https://github.com/kudrykv/latex-yearly-planner/files/7838712/sn_a5x.mos.lined.default.ampm.sun.dailycal.2022.0718.pdf)

0617 — hours are from 6 AM to 5 PM
0718 — hours are from 7 AM to 6 PM

**Reply by andytrx** [reply]
Hello,
Would you be able to correct the time on the file, sn_a5x.breadcrumb.lined.default.ampm.sun.dailycal.2022.0718.pdf
I really like what you have done with all the planners.
Thank you. Andy

**Reply by ariccb** [reply]
Hi, I found the [sn_a5x.mos.lined.default.ampm.sun.dailycal.2022.0718.pdf](https://github.com/kudrykv/latex-yearly-planner/files/7838712/sn_a5x.mos.lined.default.ampm.sun.dailycal.2022.0718.pdf) very useful, and wondering if you would be able to make 2023 and 2024 versions of that? My day also starts earlier and don't need time later than 6pm in my daily view. 

One question I have though - How difficult is it two swap which side the notes/priorities and schedule/mini month views are on? I find writing notes much more comfortable on the left side of my device, so that my hand rests more fully on the screen instead of hanging off the side.

**Reply by kudrykv** [reply]
Hi @ariccb. Sure, I'll generate one soon.
I'll look into switching columns, it should be easy enough.

**Reply by kudrykv** [reply]
@ariccb 
[sn_a5x.mos.lined.default.ampm.sun.dailycal.2023.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9253733/sn_a5x.mos.lined.default.ampm.sun.dailycal.2023.pdf)
[sn_a5x.mos.lined.default.ampm.sun.dailycal.2024.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9253734/sn_a5x.mos.lined.default.ampm.sun.dailycal.2024.pdf)

**Reply by kudrykv** [reply]
I've realized that I didn't swap columns. Let me try doing that, too.

**Reply by kudrykv** [reply]
@ariccb try these

[sn_a5x.mos.lined.default.ampm.sun.dailycal.2024.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9257191/sn_a5x.mos.lined.default.ampm.sun.dailycal.2024.pdf)
[sn_a5x.mos.lined.default.ampm.sun.dailycal.2023.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9257190/sn_a5x.mos.lined.default.ampm.sun.dailycal.2023.pdf)

**Reply by ariccb** [reply]
Wow amazing! Thanks so much! If it's easy, could you do a 2022 as well so I can finish the year with the same layout?

**Reply by kudrykv** [reply]
Sure @ariccb 
[sn_a5x.mos.lined.default.ampm.sun.dailycal.2022.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9266079/sn_a5x.mos.lined.default.ampm.sun.dailycal.2022.pdf)

**Reply by ariccb** [reply]
You're amazing. Much appreciated!!

**Comment by pbasch** [comment]
Wow! They're beautiful... though the breadcrumb all go from 9AM to 8PM.

I think I'll use the MOS dotted 0617. I wonder if I can xfer my notes
over...

*Peter Basch *

**Reply by kudrykv** [reply]
Ouch. I've edited the wrong config for the Breadcrumb, sorry :(
If you're good with MOS, then it's great and I wouldn't need to regenerate that.

>  I wonder if I can xfer my notes

Sort of. Also, prepare that your schedule will be shifted.
To transfer, copy your %pdfname%.note file and name it with %newpdfname%.note

**Comment by pbasch** [comment]
Thanks! Worked like a charm. Next time I'm in St Petersburg (or wherever)
I'll take you out for whatever one takes people out for in St Petersburg.
Or wherever.

*Peter Basch *

**Reply by kudrykv** [reply]
I'm from Ukraine ;)
Ping me if you come near Lviv, haha

**Comment by pbasch** [comment]
Life is weird, you never know.

*Peter Basch *

**Comment by andytrx** [comment]
Hello,
Would you be able to correct the time on the file, 0700>1800hrs
sn_a5x.breadcrumb.lined.default.ampm.sun.dailycal.2022.0718.pdf
I really like what you have done with all the planners.
Thank you. Andy

**Reply by kudrykv** [reply]
Hello @andytrx,
Sure. I'll generate one later today.

**Reply by kudrykv** [reply]
[sn_a5x.breadcrumb.lined.default.ampm.sun.dailycal.2022.0718.pdf](https://github.com/kudrykv/latex-yearly-planner/files/7856280/sn_a5x.breadcrumb.lined.default.ampm.sun.dailycal.2022.0718.pdf)

**Reply by andytrx** [reply]
Thank you very much.
BTW I sent you a coffee via the link above.
I really like what you have created.
I have a SN Aa5x, recently purchased and this is my first use of a digital journal.
Your design works for me.
Thank you from Australia.

**Reply by kudrykv** [reply]
Thank you!

**Reply by andytrx** [reply]
Hello again
Would you be kind enough to generate a 2023 version for me ?
sn_a5x.breadcrumb.lined.default.ampm.sun.dailycal.2023
With the time on the file, 0700>1800hrs
Thank you. Andy

**Reply by andytrx** [reply]
By the way, I have followed the instructions and installed this on Mint Linux 20.2.
It runs OK and then throws an error on the Quarterly.tex
I have attached the log. I don't know if it is something simple I am doing wrong.
[sn_a5x.breadcrumb.default.log](https://github.com/kudrykv/latex-yearly-planner/files/9993782/sn_a5x.breadcrumb.default.log)
Best Regards. Andy

**Reply by andytrx** [reply]
Hello again. Cancel that request. I fixed it myself. I didn't have the current version of Go V13 not ver 19.
Thank you for creating this.

**Comment by EdithB** [comment]
Love these so much.  I've been using them since earlier in the year and just downloaded the 2023 and 2024 versions.  They're great.

Agree with those who say that earlier starts (7 am, e.g.) would be great for a future round, but that's just a small thing.  Thank you for your work.

**Comment by pbasch** [comment]
First, I hope you're OK. God, what a world.

If you're still making Supernote planners for the bigger Supernote (5x, I
think), could I get one for 2023 and 2024?

Day from 6am to 6pm, weeks starting Sunday.

Yes, in English. Thanks!

*Peter Basch *
